dbus-codegen/src/main.rs

```diff
@@ -56,6 +58,10 @@ Defaults to 'RefClosure'."))
         let c = dbus::Connection::get_private(bus).unwrap();
         let p = c.with_path(dest, path, 10000);
         p.introspect().unwrap()
+    } else if let Some(_file_path) = matches.value_of("file")  {
+    	let mut s = String::new();
```

Comment:
This is quite redundant. Just add a `let` before the next statement to create and initialize the variable where it's needed. This is one thing that's different from C that you'll want to get used to.